lms/templates/explore_course_card.html

```diff
@@ -45,8 +45,8 @@ <h2 class="course-name">
             </h2>
             <p class="course-overview">
                 % if hasattr(course.id, 'ccx'):
-                  % if len(ccx.course_description) > 100:
-                    <span>${ccx.course_description[:100] + '...'}</span>
+                  % if len(ccx.course_description) > 120:
+                    <span>${ccx.course_description[:120] + '...'}</span>
```

Comment:
Also add same attr for affiliate and course name. Basically, wherever we trim the text.